extensions/positron-python/src/client/pythonEnvironments/creation/types.ts

```diff
@@ -13,4 +17,5 @@ export interface CreateEnvironmentOptionsInternal {
     condaPythonVersion?: string;
     uvPythonVersion?: string;
     // --- End Positron ---
+    interpreter?: string;
```

Comment:
TODO: is our `interpreterPath?` now redundant?

Reply:
Worth checking, but if there are subtle differences we can also make an issue and defer